src/inline_tests.ml

```diff
@@ -247,7 +247,8 @@ include Sub_system.Register_end_point(
           ~obj_dir
           ~modules
           ~opaque:false
-          ~requires:runner_libs
+          ~requires_compile:runner_libs
+          ~requires_link:(Lazy.from_val runner_libs)
```

Comment:
```suggestion
          ~requires_link:(lazy runner_libs)
```

Note: It is strictly better to use `lazy v` rather than `Lazy.from_val v`. They both have the same semantic, however the first form is optimised statically by the compiler. In fact, I think we should just ban uses of `Lazy.from_val` given that it is just misleading, I've never seen it used correctly.